platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -37,13 +50,18 @@ internal sealed class LoggerState {
      * The logger has been successfully started and is ready for use. Subsequent attempts to start the logger will be ignored.
      */
     class Started(
-        val logger: LoggerImpl,
+        internal val logger: ILogger,
```

Comment:
this is kinda cool, the callback now contains a fully initiated version of the ILogger which gives access to non-nullable pieces of data like the `sesssionId`, `deviceId`, etc 👍